### ISSUE TITLE
make shortcuts text more readable in shortcut select menu

### ISSUE THREAD

**Issue by Agrrox** [ickputzdirwech/Shortcuts-ick#8]
The current category prefix in the shortcut menu makes the shortcuts unreadable.
![Screen Shot 2022-01-10 at 03 37 11](https://user-images.githubusercontent.com/10901821/148713712-8e6e878c-5ae9-4df4-9f4b-10fc505a6c59.png)

Here are my suggestions: 
a) adding colored square
```
■ Deconstruction...
```
b) remove the prefix
```
Deconstruction...
```

**Comment by ickputzdirwech** [comment]
I see the issue here. I added these tags in order to make it easier to find shortcuts. (I think when I added these tags the GUI would be as wide as possible to display even the longest names.) Outright removing them is not really an option for me. Making them optional might be a solution however.

I'm sceptic about the colored square because this would be useless for colorblind people (they got at least the text right now). Also I don't know how to do this, but I guess I could use the colored signal icons.

Which made me think instead of replacing it with colored squares I could just replace it with the icon of the item the shortcut it gives you, or the shortcut is related to.

What are your thoughts on this?

PS: Just in case you don't know if you hover over the icon on the left a tooltip with the full name and the assigned controls is displayed.
